frontend/components/DocumentEditToolbar/DocumentEditToolbar.jsx

```diff
@@ -76,7 +76,7 @@ export default class DocumentListToolbar extends Component {
 
     return (
       <Toolbar>
-        {peers && peers.length &&
+        {Boolean(peers && peers.length) &&
```

Comment:
When `peers` is length 0 the condition `peers.length` is still a valid condition, meaning the output in the dom is just `0`.